pdata/internal/cmd/pdatagen/internal/base_fields.go

```diff
@@ -168,14 +168,31 @@ const accessorsOptionalPrimitiveValueTemplate = `// ${fieldName} returns the ${l
 func (ms ${structName}) ${fieldName}() ${returnType} {
 	return ms.getOrig().Get${fieldName}()
 }
+
 // Has${fieldName} returns true if the ${structName} contains a
 // ${fieldName} value, false otherwise.
 func (ms ${structName}) Has${fieldName}() bool {
 	return ms.getOrig().${fieldName}_ != nil
 }
+
 // Set${fieldName} replaces the ${lowerFieldName} associated with this ${structName}.
 func (ms ${structName}) Set${fieldName}(v ${returnType}) {
 	ms.getOrig().${fieldName}_ = &${originStructType}{${fieldName}: v}
+}
+
+// Clear${fieldName} removes the ${lowerFieldName} associated with this ${structName}.
```

Comment:
This is the only thing I want to make sure we agree on. Do we have some other similar examples?

Reply:
We have `pcommon.Map.Clear()` which is kind of similar

Reply:
We can also call it `RemoveMin`, `RemoveMax` etc, given that we have `Map.Remove` and `Slice.RemoveIf`. Let me know if you think that's a better option

Reply:
What is the different between `Map.Remove` and `Map.Clear`?

Reply:
`Map.Remove` removes a particular item under a provided key, `Map.Clear` removed all items from the map.

Reply:
Then maybe `Remove` is better.

Reply:
"Remove" seems to be removing from a collection, clear seems to make everything disappear. Kind of "ClearMin" seems that everything related to "Min" disappears.

Reply:
> "Remove" seems to be removing from a collection, clear seems to make everything disappear. Kind of "ClearMin" seems that everything related to "Min" disappears.

Yes, that is my thinking as well. In that case `HistogramDataPoint` can be a kind of a collection and Min is a field in that collection

Reply:
Given that we have `Min`, `HasMin`, and `SetMin`, I'd rather prefer `RemoveMin` than `ClearMin`. @bogdandrutu WDYT?

Reply:
Fine with "RemoveMin" please document in our name conventions :)

Reply:
@bogdandrutu, updated to Remove and added documented it in the pdata/README.md